tests/test_schunk.c

```diff
@@ -53,10 +53,27 @@ static char* test_schunk(void) {
 
   blosc2_meta_update(schunk, "metalayer2", (uint8_t *) "my metalayer2", sizeof("my metalayer2"));
 
+  char names_[2][13] = {"vlmetalayer1", "vlmetalayer2"};
+  char **names = malloc(sizeof(char**));
```

Comment:
I *think* you need to know the number of metalayers (`nvlmeta`) before the malloc (which should be `malloc(nvlmeta * sizeof(char*))`).

Reply:
Fixed in 52736067b453102049a645aa7137388b5a23e4be.